repo2docker/buildpacks/base.py

```diff
@@ -120,6 +120,12 @@
 COPY src/ ${REPO_DIR}
 RUN chown -R ${NB_USER}:${NB_USER} ${REPO_DIR}
 
+# Explicitly create $HOME as some tools like conda need it to exist
+# and will not create it if it does not exist. ${REPO_DIR} defaults
+# to ${HOME}, so this problem is hidden unless ${REPO_DIR} is set to
+# something else
+RUN mkdir -p "${HOME}" && chown ${NB_USER}:${NB_USER} "${HOME}"
```

Comment:
Yup, you're right. When we pass the right flags to adduser, this is unnecessary.

Updated the PR to the real fix, which is to avoid touching $HOME when running install-miniconda.bash as root